src/foundation.cs

```diff
@@ -3702,6 +3702,15 @@ public interface NSLocale : NSSecureCoding, NSCopying {
 		
 		[Internal, Field ("NSLocaleAlternateQuotationEndDelimiterKey")]
 		NSString _AlternateQuotationEndDelimiterKey { get; }
+
+		[Watch (3, 0), TV (10, 0), Mac (10, 12), iOS (10, 0)]
+		[Export ("calendarIdentifier")]
+		NSString CalendarIdentifier { get; }
+
+		[Watch (3,0), TV (10,0), Mac (10,12), iOS (10,0)]
+		[Export ("localizedStringForCalendarIdentifier:")]
+		[return: NullAllowed]
+		string LocalizedStringForCalendarIdentifier (NSString calendarIdentifier);
```

Comment:
`LocalizedStringForCalendarIdentifier` -> `GetLocalizedString` : adding prefix for action/verb, removing suffix that the parameter already covers